virtual-organizations/BNL.yaml

```
@@ -30,10 +30,9 @@ ID: 114
 LongName: Brookhaven National Laboratory
 OASIS:
   Managers:
-    Jason Federico:
+    John S. De Stefano Jr.:
       DNs:
-      - /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=People/CN=Jason Federico 2576
-      - /DC=com/DC=DigiCert-Grid/O=Open Science Grid/OU=People/CN=Jason Federico 2576
+      -  /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=destefan/CN=665120/CN=John Steven De Stefano Jr
       ID: 9ad4d5aa3825356e2329e7fcb97c5b2fa2014b9e
```

@deesto if you accept this suggestion I think this PR looks pretty good

```suggestion
      ID: 8a7d75ea65a2962520279197c72df6a16e1533a1
```

---

Also, are we purposefully removing Jason Federico?

---

Yes, that's fine, thanks